### PR TITLE
ADD: Add a post on xradar

### LINKED ISSUE #389: Xarray for radar analysis
This is a proposal for a radar data analysis use case story for Xarray. The story will be published on Xarray's blog (https://xarray.dev/blog). 

### Why?

One or more user stories that highlight how Xarray is used for radar data analysis would help show Xarray's impact in this area. Ideally we show how Xarray is used with radar data in operations and for basic research, as well as how it is used in the cloud and on HPCs.

### What?

We are targeting a short and non-technical post that illustrates how Xarray is used in the climate context. Below is a template outline that you should feel free to modify:

1. Who am I?
2. What problem am I trying to solve?
3. How does Xarray help?
4. Why did I choose Xarray? 
5. Current pain points (or ways that Xarray could better serve your use case)
6. Technology I use around Xarray
7. Anything else to know?
8. Links and references

Feel free to insert images or short code blocks if they help you tell your story.

### How?

Xarray's blog using Markdown with some front-matter. You can copy one of the [existing posts](https://github.com/xarray-contrib/xarray.dev/tree/main/src/posts) to get started or you can write in a google-doc-like-thing and ask us to do the markdown formatting. If you copy a previous post, use a name like `user-story-pangeo-ocean-model` or whatever makes sense for your application.

xref: https://github.com/xarray-contrib/xarray.dev/issues/272

### PR DESCRIPTION
**Pull request by mgrover1**
This introduces [xradar](https://github.com/openradar/xradar), which is a tool for working with radar data using the xarray stack.

Closes #389

### CODE REVIEW

**Comment by mgrover1** [comment]
@dcherian - this is a first draft of the radar data with xarray post @kmuehlbauer and I have been working on!

**Comment by mgrover1** [comment]
@andersy005 any chance you could enable the vercel check here? :)

**Comment by kmuehlbauer** [comment]
Thanks @andersy005 for letting it run.

I'll add the ODIM part later this week, hopefully freeing up some time.

**Comment by jhamman** [comment]
Very cool @mgrover1 - One comment. The datatree view in the [open-an-entire-volume-of-cfradial1-data](https://xarray-r2w5iooyq-xarray.vercel.app/blog/introducing-xradar#open-an-entire-volume-of-cfradial1-data) is very long. Do you think there is a way to abbreviate this without changing the message?

@TomNicholas, would be so cool if there was an interactive html repr here 😉

**Comment by jhamman** [comment]
I suspect we'll need a bit of help from @andersy005 but we _should_ be able to drop html content directly into this post. 

```python
print(dt._repr_html_())
```

will print out the HTML used in the interactive repr.

**Comment by kmuehlbauer** [comment]
> I suspect we'll need a bit of help from @andersy005 but we _should_ be able to drop html content directly into this post.
> 
> ```python
> print(dt._repr_html_())
> ```
> 
> will print out the HTML used in the interactive repr.

I'd give this a try while working on my additions, but having a hard time to figure out how to put that `div` into the markdown source.

**Comment by kmuehlbauer** [comment]
> I'd give this a try while working on my additions, but having a hard time to figure out how to put that `div` into the markdown source.

I've found a possible solution using iframe embedded html. Unfortunately I can't push to @mgrover1's branch. I'll issue a PR at his fork, to get this in.

**Comment by kmuehlbauer** [comment]
https://github.com/mgrover1/xarray.dev/pull/2

**Comment by mgrover1** [comment]
Thanks @kmuehlbauer - I merged in your suggestions!

**Comment by kmuehlbauer** [comment]
If someone can let run the vercel build we could have some more eyes to look on it. @mgrover1 You might show it today at the meeting, which I'm not able to make today.

File: src/posts/introducing-xradar/index.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+radar['sweep_0']
+```
+
+<iframe
+  width='100%'
+  height='500'
+  src='/posts/introducing-xradar/out03.html'
+  frameborder='0'
+  allowfullscreen
+></iframe>
```
Comment (jhamman) [inline src/posts/introducing-xradar/index.md:157]: This isn't a bad solution. @andersy005 - do you think we can make this less painful by following a similar approach to Pangeo-forge: 

https://github.com/pangeo-forge/pangeo-forge.org/blob/97efc8292d04cf973c60020588b40c140de38557/src/components/dataset/repr.js#LL41C14-L41C37

Comment (kmuehlbauer) [reply src/posts/introducing-xradar/index.md:157]: I did not try harder, but one remaining problem is the fixed height. I could not yet get it to expand the window on display of variables for instance.

**Comment by kmuehlbauer** [comment]
@mgrover1 Is this already mature enough to be published? It would be great to have this on the road before the conferences.

**Comment by mgrover1** [comment]
@kmuehlbauer I think so!! Not sure if we wanted to add the rest of the contributors to the package first

**Comment by kmuehlbauer** [comment]
Yes, we should definitely add all contributors.

**Comment by mgrover1** [comment]
@andersy005 can you give the all clear to re-render?

**Comment by mgrover1** [comment]
I think this is good to merge @andersy005 @kmuehlbauer

**Comment by kmuehlbauer** [comment]
> I think this is good to merge @andersy005 @kmuehlbauer 

Great!

**Comment by kmuehlbauer** [comment]
@mgrover1 Any idea on the vercel issue?

**Comment by mgrover1** [comment]
We just authorization again

**Comment by andersy005** [comment]
> I think this is good to merge @andersy005 @kmuehlbauer

apologies, folks. i was offline for the last couple of days.

File: src/posts/introducing-xradar/index.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+---
+title: 'Weather Radar Data in Xarray'
+date: '2023-05-16'
```
Comment (andersy005) [inline src/posts/introducing-xradar/index.md:3]: @mgrover1, when you get a moment can you add a `summary` entry (one sentence) here?

**Comment by mgrover1** [comment]
> > I think this is good to merge @andersy005 @kmuehlbauer
> 
> apologies, folks. i was offline for the last couple of days.

No worries! Just getting back to this now 😄 thanks for all your help here

File: src/posts/introducing-xradar/index.md (hunk 2 above)
Comment (mgrover1) [reply src/posts/introducing-xradar/index.md:3]: Done!

Comment (andersy005) [reply src/posts/introducing-xradar/index.md:3]: perfect!

Comment (kmuehlbauer) [reply src/posts/introducing-xradar/index.md:3]: Thanks @andersy005 also for the RawHtml update.🙏

**Comment by mgrover1** [comment]
@andersy005 - good to merge?

**Comment by andersy005** [comment]
thank you very much for your patience, @mgrover1 / @kmuehlbauer. yes, this is ready (merging shortly)

@jhamman, thank you for the feedback!

**Pull request merged**